README.rdoc

```diff
@@ -30,6 +30,17 @@ With GCS Remote test:
 
     GCS_PROJECT_ID="<project_id>" GCS_KEY_FILEPATH="<path_to_key>" GCS_BUCKET="<bucket_name>" bundle exec rake spec
 
+== Releasing
+
+Releasing is handled by Shopify's internal Shipit server.
+
+After merging a change, a separate PR should be opened to update `History.md` and bump the version
+(following [Semantic Versioning](https://semver.org/))
+
+When merged, the new release will be automatically tagged and deployed to rubygems.org.
+
+Note that the automatic process does not create a new release on GitHub.
```

Comment:
@max611 I noticed there's a release for your recent change, did you manually create that on GitHub?

Reply:
I honestly don't remember 100% but I do think I did it manually since I was not sure if ship it was going to do it all.